app/lib/payment_gateways/buyer_references.rb

```diff
@@ -1,34 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This file shouldn't be deleted, I think.
The method is still used by stripe: https://github.com/3scale/porta/blob/master/app/lib/payment_gateways/stripe_crypt.rb#L55